CMakeLists.txt

```diff
@@ -321,6 +321,20 @@ if(${IREE_BUILD_COMPILER} OR
   find_package(Python3 COMPONENTS Interpreter REQUIRED)
 endif()
 
+#-------------------------------------------------------------------------------
+# Check if git submodules have been initialized.
+# This will only run if python/python3 is available
+#-------------------------------------------------------------------------------
+
+execute_process(
+  COMMAND python --version && python scripts/git/check_submodule_init.py
```

Comment:
This fails if python is not available, when it should skip.

You could first check for python, or flip the check a bit like `!(python --version) || python ...` (I think that does what we want)

Reply:
Okay @ScottTodd , doing the changes.

Reply:
Done